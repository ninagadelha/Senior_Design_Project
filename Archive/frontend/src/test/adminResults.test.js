import renderer from "react-test-renderer";
import AdminResults from "../routes/adminResults";

jest.mock("react-router-dom");
jest.mock("../components/AdminNavBar");
jest.mock("../components/footer");
jest.mock("axios");
jest.mock("../styles/adminResults.css");

const renderTree = (tree) => renderer.create(tree);
describe("<AdminResults>", () => {
  it("should render component", () => {
    expect(renderTree(<AdminResults />).toJSON()).toMatchSnapshot();
  });
});
