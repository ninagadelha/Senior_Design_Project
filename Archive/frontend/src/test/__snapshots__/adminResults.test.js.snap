// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AdminResults> should render component 1`] = `
<div
  class="cont"
>
  <div
    class="content-placeholder-results"
  >
    <h1>
      Admin Results
    </h1>
    <div>
      <label>
        Enter Student Email: 
        <input
          onChange={[Function]}
          type="email"
          value=""
        />
      </label>
      <button
        className="action-button-results"
        onClick={[Function]}
      >
        Retrieve by Email
      </button>
    </div>
    <div>
      <label>
        Select a Program: 
        <select
          onChange={[Function]}
        >
          <option
            disabled={false}
            value=""
          >
            Please select a program name
          </option>
        </select>
      </label>
      <button
        className="action-button-results"
        disabled={true}
        onClick={[Function]}
      >
        Retrieve by Program Name
      </button>
    </div>
    <p>
      Please allow a few seconds for the download to start after clicking the button. If the download still does not start, please ensure that you have correctly entered the email or selected a program name.
    </p>
  </div>
</div>
`;
